### PR TITLE
fix: Change span id

### PR DESCRIPTION
**Pull request by kosssi**
ID attribute values must be unique

https://dequeuniversity.com/rules/axe/3.5/duplicate-id

### CODE REVIEW

**Comment by kosssi** [comment]
Thanks for this shortcode ;)

**Comment by martignoni** [comment]
I'd be glad to merge your PR, but I'm afraid I don't understand why the ID attribute is not unique (before your PR). Can you please elaborate?

**Comment by kosssi** [comment]
@martignoni In your script you can see two same id :
- [`id="id-{{ $fingerprint }}"></span>&#32;`](https://github.com/martignoni/hugo-cloak-email/blob/master/layouts/shortcodes/cloakemail.html#L20)
- [`<script id="id-{{ $fingerprint }}">`](https://github.com/martignoni/hugo-cloak-email/blob/master/layouts/shortcodes/cloakemail.html#L22)

Sorry I'm not explain where is.

**Comment by martignoni** [comment]
Undestood, you're right. Thanks for the fix.

File: layouts/shortcodes/cloakemail.html [hunk 1]
```diff
@@ -11,13 +11,13 @@
 {{- $fingerprint := md5 (print (.Get "address") $protocol (index (seq 999 | shuffle) 0)) | truncate 8 "" -}}
 {{/* Set via CSS what is displayed when Javascript is disabled. Query is never displayed */}}
 <style type="text/css">
-  #id-{{ $fingerprint }}.cloaked-e-mail:before {
+  #wjs-{{ $fingerprint }}.cloaked-e-mail:before {
```
Comment (martignoni) [inline layouts/shortcodes/cloakemail.html]:
```suggestion
  #span-{{ $fingerprint }}.cloaked-e-mail:before {
```

File: layouts/shortcodes/cloakemail.html [hunk 2]
```diff
@@ -11,13 +11,13 @@
...
     content:{{ with $domain }}attr(data-domain) "\0040" {{ end }}attr(data-user);
     unicode-bidi:bidi-override;
     direction:rtl;
   }
 </style>
-&#32;<span class="cloaked-e-mail" data-user="{{ range $index := seq (sub (len $user) 1) 0}}{{ substr $user $index 1}}{{ end }}"{{ with $domain }} data-domain="{{ range $index := seq (sub (len $domain) 1) 0}}{{ substr $domain $index 1}}{{ end }}"{{ end }} id="id-{{ $fingerprint }}"></span>&#32;
+&#32;<span class="cloaked-e-mail" data-user="{{ range $index := seq (sub (len $user) 1) 0}}{{ substr $user $index 1}}{{ end }}"{{ with $domain }} data-domain="{{ range $index := seq (sub (len $domain) 1) 0}}{{ substr $domain $index 1}}{{ end }}"{{ end }} id="wjs-{{ $fingerprint }}"></span>&#32;
```
Comment (martignoni) [inline layouts/shortcodes/cloakemail.html]:
```suggestion
&#32;<span class="cloaked-e-mail" data-user="{{ range $index := seq (sub (len $user) 1) 0}}{{ substr $user $index 1}}{{ end }}"{{ with $domain }} data-domain="{{ range $index := seq (sub (len $domain) 1) 0}}{{ substr $domain $index 1}}{{ end }}"{{ end }} id="span-{{ $fingerprint }}"></span>&#32;
```

**Review by martignoni — CHANGES_REQUESTED** [review]
Suggesting some better ID names.

You could change the other ID too, eg. `"script-{{ $fingerprint }}"` instead of `"id-{{ $fingerprint }}"` (lines 22-23).

**Comment by kosssi** [comment]
Done ;)

**Review by martignoni — APPROVED** [review]

**Comment by martignoni** [comment]
Thank you!

**Pull request merged**